docker/README.md

```diff
@@ -7,11 +7,14 @@ Initial setup
    * Install [Docker](https://docs.docker.com/engine/installation/)
    * Install [Docker Compose](https://docs.docker.com/compose/install/) (Note you can also install in a virtualenv with `$ pip install docker-compose`)
 * OS X
-   * Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/). Go through the full tutorial, which will create a default machine.
-   * To create a new VM manually, run `docker-machine create default --driver=virtualbox` (not necessary if you followed the Docker Toolbox tutorial).
-   * If not using the Quick Start terminal, run `eval $(docker-machine env default)` to set up Docker's environment variables.
+   * Preferred: install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/).
+   * Alternate for older Macs that do not support Docker for Mac:
+     * Install [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/). Go through the full tutorial, which will create a default machine.
+     * To create a new VM manually, run `docker-machine create default --driver=virtualbox` (not necessary if you followed the Docker Toolbox tutorial).
+     * If not using the Quick Start terminal, run `eval $(docker-machine env default)` to set up Docker's environment variables.
+
 * If you have any HQ services currently running (couch, postgres, redis, etc.), you should stop them now. 
-* Bootstrap the setup:
+* Bootstrap the setup. Skip this step if you came here from the [CommCare HQ README](https://github.com/dimagi/commcare-hq/blob/master/README.md#setup-localsettings).
```

Comment:
Maybe just clarify this and say `Skip this step and go to 'Configure your localsettings' below`